srcpkgs/bcftools/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'bcftools'
+pkgname=bcftools
+version=1.13
+revision=1
+build_style=gnu-configure
+configure_args="--with-htslib=system"
+makedepends="htslib-devel zlib-devel gsl-devel perl"
+short_desc="Read/write BCF2/VCF/gVCF files and call/filter/sum SNP and short indels"
+maintainer="Saba Gogichaishvili <sd.gogich@gmail.com>"
+license="MIT"
+homepage="https://www.htslib.org/"
+distfiles="https://github.com/samtools/${pkgname}/releases/download/${version}/${pkgname}-${version}.tar.bz2"
```

Comment:
Please expand `pkgname` at least before "releases"

Reply:
Sure thing, but why?

Reply:
* it doesn't usually save any characters
* if $pkgname changes for some reason (like libhandy and libhandy1 which is 1 library in different versions), we might want to keep the same distfiles URL

and probably more reasons I don't remember now

Reply:
Should I change all occurrences of $pkgname then? What about $version?

Reply:
yes, please remove all mentions of $pkgname in $distfiles

When updating a package, we want to keep using the same distfiles string, just bump the version and fix the checksum, so having only one place where we need to change the version is very useful, don't remove it.